planpro_importer/planpro110/reader110.py

```diff
@@ -109,30 +117,41 @@ def _get_next_edge(_last_node_uuid, _second_last_node):
                     completed = False
                     break
                 second_last_node_uuid = last_node_uuid
-                length_remaining = length_remaining - next_edge.GEO_Kante_Allg.GEO_Laenge.Wert
+                length_remaining = length_remaining - float(
+                    next_edge.GEO_Kante_Allg.GEO_Laenge.Wert
+                )
                 last_node_uuid = _get_other_uuid(second_last_node_uuid, next_edge)
 
             if completed:
                 edge = Edge(node_a, node_b, length=length, uuid=top_kante_uuid)
                 edge.intermediate_geo_nodes = geo_nodes_in_order
                 self.topology.add_edge(edge)
             else:
-                print(f"Warning: TOP_EDGE {top_kante_uuid} could not be completed, "
-                      f"since the chain of geo edges is broken after {last_node_uuid}. "
-                      f"This may cause errors later, since the topology is broken.")
+                print(
+                    f"Warning: TOP_EDGE {top_kante_uuid} could not be completed, "
+                    f"since the chain of geo edges is broken after {last_node_uuid}. "
+                    f"This may cause errors later, since the topology is broken."
+                )
 
     def read_signals_from_container(self, container):
         for signal in container.Signal:
             if signal.Signal_Real is None:
                 continue
-            if len(signal.Punkt_Objekt_TOP_Kante) != 1:  # If other than 1, no real signal with lights
+            if (
+                len(signal.Punkt_Objekt_TOP_Kante) != 1
+            ):  # If other than 1, no real signal with lights
                 continue
-            if signal.Bezeichnung is None or signal.Bezeichnung.Bezeichnung_Aussenanlage is None:
+            if (
```

Comment:
FYI, if we want to pylint this code at some point, [PEP 8 prefers a different style of breaking long lines in if statements](https://peps.python.org/pep-0008/) (ctrl+f `if (`).

Reply:
👍